Travel Scheduler/View Controllers/MoreOptionViewController.m

```diff
@@ -29,12 +37,102 @@ - (void)viewDidLoad
     [self.view addSubview:label];
     [self.collectionView reloadData];
-    self.scheduleButton = makeButton(@"Generate Schedule", CGRectGetHeight(self.view.frame), CGRectGetWidth(self.view.frame), self.collectionView.frame.origin.y + CGRectGetHeight(self.collectionView.frame));
+    self.scheduleButton = makeButton(@"Generate Schedule", CGRectGetHeight(self.view.frame), CGRectGetWidth(self.view.frame), self.collectionView.frame.origin.y-47 + CGRectGetHeight(self.collectionView.frame));
     [self.scheduleButton addTarget:self action:@selector(makeSchedule) forControlEvents:UIControlEventTouchUpInside];
     [self.view addSubview:self.scheduleButton];
+    [self createMoreOptionSearchBar];
+    [self.view addSubview:self.moreOptionSearchBarAutoComplete];
+    [self.view addSubview:self.scheduleButton];
+    self.moreOptionSearchBarAutoComplete.delegate = self;
+    [self.view addSubview:self.searchButton];
+}
+
+#pragma mark - GMSAutocomplete set up
+
+- (void) createFilterForGMSAutocomplete
+{
+    CLLocationCoordinate2D neBoundsCorner = CLLocationCoordinate2DMake(-33.843366, 151.134002);
+    CLLocationCoordinate2D swBoundsCorner = CLLocationCoordinate2DMake(-33.875725, 151.200349);
+    GMSCoordinateBounds *bounds = [[GMSCoordinateBounds alloc] initWithCoordinate:neBoundsCorner coordinate:swBoundsCorner];
+    GMSAutocompleteFilter *filter = [[GMSAutocompleteFilter alloc] init];
+    filter.type = kGMSPlacesAutocompleteTypeFilterCity;
+    _fetcher = [[GMSAutocompleteFetcher alloc] initWithBounds:bounds filter:filter];
+    _fetcher.delegate = self;
+}
+
+- (void)didAutocompleteWithPredictions:(NSArray *)predictions
+{
+    self.resultsArr = [[NSMutableArray alloc] init];
+    for (GMSAutocompletePrediction *prediction in predictions) {
+        [self.resultsArr addObject:[prediction.attributedFullText string]];
+    }
+}
+
+- (void)didFailAutocompleteWithError:(NSError *)error
+{
+    NSString *errorMessage = [NSString stringWithFormat:@"%@", error.localizedDescription];
+    NSLog(@"%@", errorMessage);
+}
+
+
+#pragma  mark - creating search bar
+
+- (void)createMoreOptionSearchBar
+{
+    CGRect screenFrame = self.view.frame;
+    self.moreOptionSearchBarAutoComplete = [[UISearchBar alloc] initWithFrame:CGRectMake(5, 150, CGRectGetWidth(screenFrame) - 10, CGRectGetHeight(screenFrame)-850)];
+    self.moreOptionSearchBarAutoComplete.backgroundColor = [UIColor blackColor];
+    self.moreOptionSearchBarAutoComplete.autocorrectionType = UITextAutocorrectionTypeNo;
+    self.moreOptionSearchBarAutoComplete.autocapitalizationType = UITextAutocapitalizationTypeNone;
+    self.moreOptionSearchBarAutoComplete.autoresizingMask = UIViewAutoresizingFlexibleWidth;
+    self.moreOptionSearchBarAutoComplete.backgroundColor = [UIColor whiteColor];
+    self.moreOptionSearchBarAutoComplete.searchBarStyle = UISearchBarStyleMinimal;
+    self.moreOptionSearchBarAutoComplete.placeholder = @"Search destination of choice...";
```

Comment:
Were we supposed to delete the ellipsis?

Reply:
I think it would be a good idea to delete it since it's kind of unusual to see that in an app's search bar.